content/guides/4.connect/2.filter-rules.md

```diff
@@ -120,6 +141,7 @@ This behavior can be overridden by using the explicit `_some` and `_none` operat
   }
 }
 ```
+In the above, `categories` is the relational field in the collection. `_none` checks if no related category has the name "Recipe". If an article has no categories named "Recipe", it will be included in the result.
```

Comment:
```suggestion
In the above, `categories` is the relational field in the collection. `_none` checks if no related category has the name "Recipe". If an item has no categories named "Recipe", it will be included in the result.
```